### DISCUSSION TITLE
Frigate in docker and home assistant in docker

### DISCUSSION THREAD

**Discussion by martinusdepinus** [blakeblackshear/frigate#7400] [Ask A Question]
Hello to all,

In home assistant (within a docker) there are no add ons. 
Not for as far as i can find them :-).

So i have frigate running in a docker on my synology dsm7 and home assitant running in a docker.

How do i get de frigate inside the home assitent.?

Hope that someone has a answer for me

thenk in advanced

**Comment by NickM-27** [comment]
You install HACS and then install the frigate integration
